tests/pages/pix-pro/index.test.js

```diff
@@ -37,22 +37,62 @@ describe('Index Page', () => {
       })
     })
 
-    describe('when there is a cookie', () => {
-      test('redirects to locale page', async () => {
-        // given
-        document.cookie = 'foo=bar; locale=fr'
+    describe('when there is a locale cookie', () => {
+      describe('with international format', () => {
```

Comment:
```suggestion
      describe('with value being only one subtag', () => {
```

cf. https://www.rfc-editor.org/rfc/rfc5646.html

Reply:
Traité 😸

Reply:
Ce point de renommage du `describe` n'a pas été traité ou bien est-ce que je me trompe ?

Reply:
Tu ne te trompes pas 😆 
C'est corrigé !